### DISCUSSION TITLE
Django Test client with with async resolvers

### DISCUSSION THREAD

**Discussion by JanoschDeurer** [mirumee/ariadne#518] [Q&A] [answered]
Hey, thanks for the great library :) I'm moving my project from an sqlalchemy based backend to django. I'm having problems with the async resolvers and the django test client. If I write async test methods and use the async test client of django I get the following error message:
```
RuntimeError: There is no current event loop in thread 'ThreadPoolExecutor-0_0'.
```
When I use the normal django test client I get the following:
```
RuntimeError: GraphQL execution failed to complete synchronously.
```
Does anyone have any ideas on how to get around this?

**Comment by markedwards** [comment, marked as answer]
See this thread: https://github.com/mirumee/ariadne/discussions/493

**Reply by JanoschDeurer** [reply]
Thanks I'll keep following that :)
